src/Codeception/Module/Laravel5.php

```diff
@@ -1167,6 +1167,63 @@ public function haveMultiple($model, $times, $attributes = [], $name = 'default'
             $this->fail("Could not create model: \n\n" . get_class($e) . "\n\n" . $e->getMessage());
         }
     }
+    
+    /**
+     * Use Laravel's model factory to make a model instance.
+     * Can only be used with Laravel 5.1 and later.
+     *
+     * ``` php
+     * <?php
+     * $I->make('App\User');
+     * $I->make('App\User', ['name' => 'John Doe']);
+     * $I->make('App\User', [], 'admin');
+     * ?>
+     * ```
+     *
+     * @see http://laravel.com/docs/5.1/testing#model-factories
```

Comment:
I would ask you to write a bit more how data factories are defined. Link to laravel website is not enough as it doesn't say how to use it with codeception